ru-intl/remap/en/background.js

```diff
@@ -0,0 +1,199 @@
+                          "a": "á", "A": "Á", "n": "ń", "N": "Ń", "i": "í", "I": "Í", "w": "ẃ", "W": "Ẃ",
+                          " ": " ́", "z": "ź", "Z": "Ź", "r": "ŕ", "R": "Ŕ", "m": "ḿ", "M": "Ḿ", "p": "ṕ",
+                          "P": "Ṕ", "s": "ś", "S": "Ś", "g": "ǵ", "G": "Ǵ", "k": "ḱ", "K": "Ḱ", "l": "ĺ",
+                          "L": "Ĺ", "c": "ć", "C": "Ć", },
+    "dead_grave"      : { "u": "ù", "U": "Ù", "y": "ỳ", "Y": "Ỳ", "o": "ò", "O": "Ò", "e": "è", "E": "È",
+                          "a": "à", "A": "À", "n": "ǹ", "N": "Ǹ", "i": "ì", "I": "Ì", "w": "ẁ", "W": "Ẁ",
+                          " ": "`", },
+    "dead_diaeresis"  : { "u": "ü", "U": "Ü", "y": "ÿ", "Y": "Ÿ", "o": "ö", "O": "Ö", "e": "ë", "E": "Ë",
+                          "a": "ä", "A": "Ä", "i": "ï", "I": "Ï", "w": "ẅ", "W": "Ẅ", " ": "¨", },
+    "dead_tilde"      : { "u": "ũ", "U": "Ũ", "y": "ỹ", "Y": "Ỹ", "o": "õ", "O": "Õ", "e": "ẽ", "E": "Ẽ",
+                          "a": "ã", "A": "Ã", "n": "ñ", "N": "Ñ", "i": "ĩ", "I": "Ĩ", " ": "~", },
+    "dead_circumflex" : { "u": "û", "U": "Û", "y": "ŷ", "Y": "Ŷ", "o": "ô", "O": "Ô", "e": "ê", "E": "Ê",
+                          "a": "â", "A": "Â", "i": "î", "I": "Î", "w": "ŵ", "W": "Ŵ", " ": "^",
+                          "z": "ẑ", "Z": "Ẑ", "s": "ŝ", "S": "Ŝ", "g": "ĝ", "G": "Ĝ", "c": "ĉ", "C": "Ĉ", },
+    "dead_breve"      : { "u": "ŭ", "U": "Ŭ", "o": "ŏ", "O": "Ŏ", "e": "ĕ", "E": "Ĕ", "a": "ă", "A": "Ă",
+                          "g": "ğ", "G": "Ğ", "i": "ĭ", "I": "Ĭ", " ": "˘", },
+    "dead_caron"      : { "u": "ǔ", "U": "Ǔ", "o": "ǒ", "O": "Ǒ", "e": "ě", "E": "Ě", "a": "ǎ", "A": "Ǎ",
+                          "n": "ň", "N": "Ň", "i": "ǐ", "I": "Ǐ", " ": "ˇ", "z": "ž", "Z": "Ž", "r": "ř",
+                          "R": "Ř", "s": "š", "S": "Š", "g": "ǧ", "G": "Ǧ", "k": "ǩ", "K": "Ǩ", "l": "ľ",
+                          "L": "Ľ", "c": "č", "C": "Č", },    
+    "dead_doubleacute": { "u": "ű", "U": "Ű", "o": "ő", "O": "Ő", " ": "˝", },
+    "dead_ogonek"     : { "u": "ų", "U": "Ų", "o": "ǫ", "O": "Ǫ", "e": "ę", "E": "Ę", "a": "ą", "A": "Ą",
+                          "i": "į", "I": "Į", " ": "˛", },
+    "dead_macron"     : { "u": "ū", "U": "Ū", "y": "ȳ", "Y": "Ȳ", "o": "ō", "O": "Ō", "e": "ē", "E": "Ē",
+                          "a": "ā", "A": "Ā", "i": "ī", "I": "Ī", " ": "¯", },
+    "dead_abovedot"   : { "y": "ẏ", "Y": "Ẏ", "o": "ȯ", "O": "Ȯ", "e": "ė", "E": "Ė", "a": "ȧ", "A": "Ȧ",
+                          "n": "ṅ", "N": "Ṅ", "i": "ı", "I": "İ", "w": "ẇ", "W": "Ẇ", " ": "˙",
+                          "z": "ż", "Z": "Ż", "r": "ṙ", "R": "Ṙ", "m": "ṁ", "M": "Ṁ", "p": "ṗ", "P": "Ṗ",
+                          "s": "ṡ", "S": "Ṡ", "g": "ġ", "G": "Ġ", "c": "ċ", "C": "Ċ", },
+    "dead_abovering"  : { "u": "ů", "U": "Ů", "y": "ẙ", "a": "å", "A": "Å", "w": "ẘ", " ": "°", },
+    "dead_cedilla"    : { "c": "ç", "C": "Ç", "s": "ş", "S": "Ş", "t": "ţ", "T": "Ţ", " ": "¸", },
+};
+
+var altstate = false;
```

Comment:
Would it be possible to replace var with let everywhere?

Reply:
Done